modules/math/src/main/java/com/opengamma/strata/math/impl/interpolation/data/InterpolationBoundedValues.java

```diff
@@ -13,14 +13,22 @@
  * 
  */
 public final class InterpolationBoundedValues implements Serializable {
+
   private static final long serialVersionUID = 1L;
+
   private final int _lowerBoundIndex;
-  private final Double _lowerBoundKey;
-  private final Double _lowerBoundValue;
+  private final double _lowerBoundKey;
+  private final double _lowerBoundValue;
   private final Double _higherBoundKey;
```

Comment:
Those fields are optional. I didn't want to break too much, so left them as is. Its not the best design anyway...